Interfaces/API/Dirac.py

```diff
@@ -872,8 +872,9 @@ def runLocal(self, job):
           elif os.path.isdir( isFile ):
             self.log.debug("Input Sandbox %s is a directory, found in the user working directory, copying it" % isFile)
             shutil.copytree( isFile, os.path.basename( isFile ), symlinks = True )
-          elif os.path.exists( isFile ):
-            self.log.debug("Input Sandbox %s is a file, found in the current working directory, copying it" % isFile)
+          elif os.path.exists(os.path.join(curDir, os.path.basename(isFile))):
```

Comment:
Here we check that the file is in the directory where the submission is called (`curDir`), then the file is copied